build-system/tasks/bundle-size.js

```diff
@@ -188,7 +198,7 @@ async function skipBundleSize() {
 async function reportBundleSize() {
   if (isTravisPullRequestBuild()) {
     const baseSha = gitTravisMasterBaseline();
-    const bundleSize = parseFloat(getGzippedBundleSize());
+    const bundleSize = parseFloat(getBundleSize('gzip'));
```

Comment:
Worth adding a TODO here to change `gzip` to `brotli`?

Reply:
Done.